First commit for new Community Theme #501

Timo-Breumelhof opened the pull request:
### Description of PR...
First commit for the new Community Theme.
Not perfect yet, but let's start testing report issues and I'll fix them...

## Changes made
- Named the community theme community-default
- Built e new Theme ;-)

## How did you test these updates?  
<!-- Please be as descriptive as possible. -->
On a local installation

## PR Template Checklist

- [ ] Fixes Bug
- [x] Feature solution
- [ ] Other
- [ ] Requires documentation updates  
- [ ] I've updated the documentation already  


## Please mark which issue is solved
<!-- Type numbers directly after the #, it will show the issues with that number -->

Close #501

johnhenley commented:
@Timo-Breumelhof 
1. Are you waiting for one (or both) of us to review and merge this?
2. It looks this PR is now naming the theme "community-default" rather than "community"? If so, we need to adjust the config file that generates/installs during a new install, since that now uses "community".
3. It looks like this PR is removing the existing "community" theme artifacts from the project. Was that intended? If so, it looks like the template files are also being deleted.

Timo-Breumelhof commented:
@johnhenley Well, I first wanted you to have a look at this. :-)
I did rename the Theme as I would like to do a community-bootstrap in the future next to the community-default.
What do you think of the naming?
For that one I also want to start fresh as the current one is overruling all the stuff we removed from module.css (it's messy because of that). This is why I removed it for now and then I'll work on a BS5 version for a next version?
Does that make sense to you?
